jekyll/_cci2/orb-author-intro.md

```diff
@@ -1,95 +1,72 @@
 ---
 layout: classic-docs
-title: "Introduction to Authoring an Orb"
+title: "Introduction to Authoring Orbs"
 short-title: "Authoring Orbs Introduction"
 description: "Starting point for how to authoring an orb"
 categories: [getting-started]
 order: 1
 ---
-## Introduction to Orb Authoring
 
-Orbs are reusable packages of CircleCI configuration that you may share across projects, enabling you to create encapsulated, parameterized commands, jobs, and executors that can be [used across multiple projects]({{ site.baseurl }}/2.0/using-orbs/). 
+* TOC
+{:toc}
 
-[Orbs]({{ site.baseurl }}/2.0/orb-intro/) are made available for use in a configuration through the `orbs` key in the top level of your 2.1 [.circleci/config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file.
+## Quick Start
 
-If you have determined that using an existing CircleCI or partner-certified orb will not address your specific workflow or job, then you may choose to author your own orb. Although more time-consuming than using an existing orb from the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/), authoring your own orb will enable you to create a pre-packaged configuration that suits your particular workflow. These pages will walk you through the steps required to author an orb, validate its configuration, and then publish this newly-created orb to the CircleCI Orb Registry.
+Orbs take [reusable configuration]({{site.baseurl}}/2.0/orb-concepts/#reusable-configuration) and package it in a way that can be published to and accessed on the [Orb Registry](https://circleci.com/orbs/registry/) so that they may be imported in multiple configuration files. If you manage multiple similar projects, consider abstracting out your config with orbs.
```

Comment:
Maybe 
```
a way that can be published to and accessed on the [Orb Registry](https://circleci.com/orbs/registry/) so that they may be imported in multiple configuration files.
|
V
a way that can be published to the [Orb Registry](https://circleci.com/orbs/registry/) and imported into multiple configuration files.
```